packages/postcss-convert-values/src/index.js

```diff
@@ -24,7 +24,12 @@ function parseWord(node, opts, keepZeroUnit) {
   const pair = unit(node.value);
   if (pair) {
     const num = Number(pair.number);
-    const u = pair.unit;
+    let u = pair.unit;
+
+    if (u[0] === '.') {
+      u = u.slice(1);
+    }
+
```

Comment:
Why removing the leading `.` ?

Reply:
my thought  is that is not the part of `unit` but the `value`

Reply:
for input like https://github.com/TrySound/postcss-value-parser/blob/master/test/unit.js#L10, there would be still number (length) in the unit. 

I wonder whether this should be fixed in `postcss-value-parser` as `.3rem` or `3rem` is even a value unit or not. 

cc @evilebottnawi

Reply:
@evilebottnawi

Reply:
Yes, it is not good idea

Reply:
@SkReD can you undo this change and we can submit a issue in the parser repo as this should be handled by the parser not by the minifier